mvpa2/tests/test_cmdline_ttest.py

```diff
@@ -0,0 +1,80 @@
+# emacs: -*- mode: python; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the PyMVPA package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""Unit tests for PyMVPA cmdline ttest"""
+
+from mvpa2.testing import *
+from mvpa2 import pymvpa_dataroot
+skip_if_no_external('h5py')
+from mvpa2.cmdline.cmd_ttest import run, guess_backend
+from mvpa2.datasets import Dataset
+from mvpa2.datasets.mri import fmri_dataset
+from mvpa2.base.hdf5 import h5save
+import sys
+from tempfile import mkdtemp
+from shutil import rmtree
+
+import numpy as np
+from mvpa2.misc.stats import ttest_1samp
+import scipy.stats as stats
+from os.path import join as pjoin
+
+if __debug__:
+    from mvpa2.base import debug
+
+def test_guess_backend():
+    assert_equal('nifti', guess_backend('meh.nii.gz'))
+    assert_equal('nifti', guess_backend('meh.nii'))
+    assert_equal('hdf5', guess_backend('meh.hdf5'))
+    assert_equal('hdf5', guess_backend('meh.h5'))
+    assert_equal('nifti', guess_backend('meh.tar'))
+
+# test that using a mask works
+datafn = pjoin(pymvpa_dataroot,
+               'haxby2001/sub001/BOLD/task001_run001/bold_25mm.nii.gz')
+maskfn = pjoin(pymvpa_dataroot,
+               'haxby2001/sub001/masks/25mm/brain.nii.gz')
+
+tmpdir = mkdtemp()
```

Comment:
@yarikoptic this tmpdir doesn't get deleted ever. can I save a new hdf5 file under pymvpa_dataroot for later reuse?